image/golem_blender_app/golem_blender_app/commands/verify.py

```diff
@@ -1,15 +1,19 @@
 from pathlib import Path
+
 from typing import Tuple, Optional
 import json
 import shutil
 import zipfile
 
+from golem_task_api.apputils.task import SubtaskStatus
```

Comment:
alphabetic ordering would be after `golem_blender_app`

Reply:
I'd leave it as is - it's similar to `from . import ...`

Reply:
alphabetic ordering does not know exceptions :P but i'm fine with leaving it as is :)